.github/workflows/playwright-ui.yml

```diff
@@ -0,0 +1,47 @@
+name: Playwright UI Tests
+
+on:
+  deployment_status:
+    states: ['success']
+
+jobs:
+  ui-tests:
+    name: Playwright UI Tests
```

Comment:
🍹 I feel specifying name is redundant here as we have only one workflow which is named the same and it looks `Playwright UI Tests / Playwright UI Tests` on the checks in a PR.